run-clippy.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+set -v
+echo "start to run cargo clippy"
+cargo clippy --fix --allow-staged >> output.txt
```

Comment:
Why don't we just print to stdout? Personally, I'd rather see issues in the console than look in a text file.

If we are to go with a test file then can we give it a more specific name and add it to .gitignore to prevent it from being committed by accident.